internal/backends/collection.go

```diff
@@ -227,11 +227,9 @@ type CollectionStatsParams struct {
 
 // CollectionStatsResult represents the results of Collection.Stats method.
 //
-// CountObjects is an estimate of the number of documents.
-//
-// TODO https://github.com/FerretDB/FerretDB/issues/2447
+// CountDocuments is an estimate of the number of documents.
```

Comment:
That comment was only needed because it wasn't clear what are "objects". I think it is already clear now

Reply:
Removed comment